### PR TITLE
Update community.json

### PR DESCRIPTION
**Pull request by kevinlindley**
Added the delinquencer and demoncore scripts

### CODE REVIEW

**Comment by kevinlindley** [comment]
Am I doing something wrong ?

File: community.json [hunk 1]
```diff
@@ -1502,6 +1502,21 @@
...
+      "tags": ["sequencer", "midi", "engine"]
+    },
+    {
+      "project_name": "demoncore",
+      "project_url": "https://github.com/kevinlindley/demoncore",
+      "author": "Kevin Lindley",
+      "description": "a simple noise module but with a mind of its own",
+      "discussion_url": "https://llllllll.co/t/demon-core-a-very-simple-noise-module-for-norns-but-with-a-mind-of-its-own/",
+      "tags": ["noise"]
+    },
```
Comment (ngwese) [inline community.json:1519]: This line needs to be followed by an open curly brace `{` for the next script

**Review by ngwese — CHANGES_REQUESTED** [review]

**Comment by ngwese** [comment]
If you are stuck let me know an I can fix it manually.

**Comment by kevinlindley** [comment]
Hi Greg,

Made the change but just can't seem to get the hang of GitHub's workflow. Using GitHub exceeds my patience threshold. :-)

Kind Regards
Kevin.

**Comment by ngwese** [comment]
so sorry to have dropped the ball on this. i got lost i my own projects in the evening and forgot to follow through on this. i’ll fix it up and push it through.

**Comment by ngwese** [comment]
okay you should be good to go now
https://github.com/monome/norns-community/commit/2a4090ff9c6bfd08b38b8ebb9599d487d420c622

again sorry for the bumps

**Comment by kevinlindley** [comment]
Thank you, problem at my end.

Kind Regards
Kevin

**Pull request closed without merging**